antarest/launcher/model.py

```diff
@@ -324,3 +344,256 @@ class LauncherLoadDTO(AntaresBaseModel, extra="forbid", alias_generator=to_camel
         description="The status of the launcher: 'SUCCESS' or 'FAILED'",
         title="Launcher Status",
     )
+
+
+class LauncherConfigDTO(AntaresBaseModel):
+    id: Optional[str] = None
+    name: str
+    linear_solver: str
+    min_antares_version: Optional[StudyVersion] = None
+    max_antares_version: Optional[StudyVersion] = None
+    linear_solver_param_optim_1: Optional[SolverParams] = None
+    linear_solver_param_optim_2: Optional[SolverParams] = None
+    linear_solver_param: Optional[SolverParams] = None
+    use_optim_1_basis_next_week: bool = True
+    use_optim_1_basis_optim_2: bool = True
+
+    @field_validator("name")
+    def name_not_empty(cls, v: str) -> str:
+        if not v.strip():
+            raise BadLauncherConfigInput("name cannot be empty or whitespace")
+        return v
+
+    @field_validator(
+        "linear_solver_param",
+        "linear_solver_param_optim_1",
+        "linear_solver_param_optim_2",
+    )
+    def validate_solver_params(cls, sp: Optional[SolverParams]) -> Optional[SolverParams]:
+        if not sp:
+            return sp
+        for k, v in sp:
+            if not ALLOWED_LAUNCHER_CONFIG_PARAM_PATTERN.match(k):
+                raise BadLauncherConfigInput(
```

Comment:
In pydantic validator, we should stick with `ValueError`, see https://docs.pydantic.dev/latest/errors/errors/ 

This will already be converted to a 400.